### PR TITLE
Save/Load Backend+Frontend integration

### PR DESCRIPTION
**Pull request by BrightSun95**
User's can copy their own session ID from the sharing tab.
User's can save CSV's to the server; the CSV's are associated with their session ID.
Still working on/deciding how to load those server-side CSV's onto the client machine.
     a) serverside PHP could make copies of the other user's CSV and name the new copy based on 
         currrent user's session ID (this allows current user to have a copy of the other user's CSV 
         associated with their own session ID)
     b) might just give client a path to the other user's CSV. This is sufficient for temporary use but
         persist across sessions. (can't save it or load it)

### CODE REVIEW

File: app/css/style.css [hunk 1]
```diff
@@ -5219,7 +5219,7 @@ small {
       display: block;
       width: 100%;
```
Comment (CaseyHillers) [inline app/css/style.css]: remove style.css from this

File: app/js/functions.js [hunk 2]
```diff
@@ -185,20 +264,15 @@ function parseHash(fieldsById) {
...
-    });
-  }
+  hashish.attr("href", function(href) {
+    return href + location.hash;
+  });
+}
 
 //Inital map setup
-var map = d3.select("#map"),
-  zoom = d3.behavior.zoom()
```
Comment (CaseyHillers) [inline app/js/functions.js]: put this back

File: app/uploader/upload-manager.php [hunk 3]
```diff
@@ -24,4 +26,24 @@
...
+    $csvName = "upload/";
+    $csvName .= $_POST['name'];
+    $session_file_string = file_get_contents($csvName);
+    return ($session_file_string);
+  }
+}
 ?>
+
+
+
```
Comment (CaseyHillers) [inline app/uploader/upload-manager.php]: why all the empty space at the end

**Review by CaseyHillers — CHANGES_REQUESTED** [review]
Uncaught TypeError: Cannot read property 'name' of undefined
    at saveCSV (functions.js:71)
    at saveSession (ui.js:65)
    at HTMLAnchorElement.onclick (index.html:98)

File: app/css/style.css (hunk 1 above)
Comment (BrightSun95) [reply app/css/style.css]: Should I delete the whole CSS folder?

File: app/js/functions.js (hunk 2 above)
Comment (BrightSun95) [reply app/js/functions.js]: Not sure why this looks like this in the pull request but it the stuff is back in if you check the branch source.

**Comment by CaseyHillers** [comment]
This got moved to pull request #26

**Pull request closed without merging**